app/src/scripts/nav/navbar.jsx

```diff
@@ -162,22 +162,54 @@ class BSNavbar extends Reflux.Component {
     } else {
       return (
         <div className="navbar-right sign-in-nav-btn">
-          <button
-            className="btn-blue half"
-            onClick={userStore.googleSignIn.bind(null)}>
-            <i className="fa fa-google" />
-            <span> Sign in</span>
-          </button>
-          <button
-            className="btn-blue half"
-            onClick={userStore.orcidSignIn.bind(null)}>
-            <span className="icon"><img alt="ORCID" width="16" height="16" src="https://orcid.org/sites/default/files/images/orcid_24x24.png" /></span>
-            <span> Sign in</span>
-          </button>
+          <div
+            className="login-nav-right"
+            onClick={this._toggleNav.bind(this, 'login')}
+            onMouseLeave={this._toggleNav.bind(this, 'login')}>
+            <a className="nav-link">
+              <span className="link-name">
+                {' '}
+                Sign in <span className="arrow" />
+              </span>
+            </a>
+            {!this.state.login && this._loginNav()}
+          </div>
         </div>
       )
     }
   }
+
+  _toggleNav(name) {
+    let newState = {}
+    newState[name] = !this.state[name]
+    this.setState(newState)
+  }
+
+  _loginNav() {
+    return (
+      <div className="dropdown-login">
+        <button
+          className="btn-blue half"
+          onClick={userStore.googleSignIn.bind(null)}>
+          <i className="fa fa-google" />
+          <span> Sign in</span>
```

Comment:
One thing I'm wondering about is if the "<icon> sign in" is clear enough what each is for. I took a look at Google's sign in button guidelines and it seems like even if we don't use their color scheme, we might want to adopt the "<icon> Sign in with Google" text and "<icon> Sign in with ORCID" versions for this menu to make each button's provider more obvious.